app/views/global_user_roles/search.html.haml

```diff
@@ -0,0 +1,20 @@
+= content_for :h1 do
+  = t("pages.global_settings")
+
+= content_for :sidebar do
+  = render "admin/shared/sidenav_global", sidenav_tab: "global_user_roles"
+
+%h2= t(".h2")
+= form_tag user_search_results_path, id: "ajax_form" do
+  %p.instructions
+    = t(".instructions.search")
+    - if SettingsHelper.feature_on?(:create_users)
+      = t(".instructions.add_user")
+
+  = text_field_tag :search_term, nil, size: 30, class: "search-query"
+  = submit_tag "Search", class: "btn"
+  = hidden_field_tag :search_type, "global_user_role"
+  = hidden_field_tag :create_link,
```

Comment:
The `_create_new_user` partial checks the hidden field AND the feature flag, so setting this to `true` here in all cases is fine, if maybe misleading. I'd like to leave the feature check here to make the intent more explicit.
Interestingly `hidden_field_tag` takes care of the stringification so no `#to_s` necessary.